src/dbc.rs

```diff
@@ -66,9 +66,15 @@ use crate::{
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub struct Dbc {
+    /// Encrypted information for and about this DBC's owner
```

Comment:
Great that you filled in these missing comments!

Definitely helps to have the main struct members described!